package.json

```diff
@@ -38,6 +41,7 @@
     "moment": "2.12.0",
     "read": "1.0.7",
     "request": "2.69.0",
+    "semver": "5.1.0",
```

Comment:
So we are emitting a warning at install time, documenting this in the README, maybe we'll document in the website soon after, do we really need to warn when starting the server too?
I mean at this point, if people don't read docs...
I would prefer to not have an extra dependency for those who do not upgrade their Node.

Of course, I won't be blocking this if you think we reeeally should, it's just my 2c. :)